jsettlers.graphics/res/layout/WarriorsLayout.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE element SYSTEM "layout.dtd">
+<element x="9" y="219" maxx="127" maxy="435">
+	<element class="Label" x="9" maxx="127" y="219" maxy="234">
+		<localized label="warriors_title" />
+		<font-size size="NORMAL" />
+	</element>
```

Comment:
How it is possible to pass the calculated value out of ManaInformation.getNextUpdateProgressPercent() to the label?